group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
```

Comment:
Only `site_hosts` get added as hosts, not the key. I had staging as `staging.example.com` but it seems like the site **name** should be consistent across all the env files. After all it's the same project. It's just the settings that vary.

This name is only descriptive and is never used as an actual "domain". The project is called "example.com".